### ISSUE TITLE
Invite links: all group admins should be able to approve group join requests

### ISSUE THREAD

**Issue by saulmc** [ephemeraHQ/converse-app#876]
This should reduce the join failure rate (and is how Signal works).

**Comment by nmalzieu** [comment]
@saulmc So there are 2 ways an admin approves a joins request
- automatically through a silent notification
- manually through the UI

I guess if you're talking of join failure rate you're mainly referring to the first one?

However isn't there a privacy issue?

Right now:
- an admin creates an invite link
- to be notified, he tells our backend, and our backend will send him a background notification 

But with that task, our backend needs to know for each group admin how to send them a notification. So our backend will know the list of admins for the group. Basically, if any admin creates an invite link, he will expose the full list of admins to our backend.

Not yet sure about how technically we would do that but still want to know if that's ok

**Comment by saulmc** [comment]
Yep, the silent notification path is still failing pretty often, which this redundancy would help to reduce. But we don't want full lists of group admins on the backend. @alexrisch did you have a different way in mind?
